homeassistant/components/lutron_caseta/cover.py

```diff
@@ -78,3 +68,79 @@ async def async_set_cover_position(self, **kwargs: Any) -> None:
         if ATTR_POSITION in kwargs:
             position = kwargs[ATTR_POSITION]
             await self._smartbridge.set_value(self.device_id, position)
+
+
+class LutronCasetaTiltOnlyBlind(LutronCasetaCoverBase):
+    """Representation of a Lutron tilt only blind."""
+
+    _attr_supported_features = (
+        CoverEntityFeature.OPEN_TILT
+        | CoverEntityFeature.CLOSE_TILT
+        | CoverEntityFeature.STOP_TILT
+        | CoverEntityFeature.SET_TILT_POSITION
+        | CoverEntityFeature.OPEN_TILT
+    )
+    _attr_device_class = CoverDeviceClass.BLIND
+
+    @property
+    def is_closed(self) -> bool:
+        """Return if the cover is closed."""
+        return self._device["tilt"] == 0 or self._device["tilt"] == 100
+
+    @property
+    def current_cover_tilt_position(self) -> int:
+        """Return the current position of cover."""
+        return self._device["tilt"]
+
+    async def async_close_cover_tilt(self, **kwargs: Any) -> None:
+        """Close the cover."""
+        await self._smartbridge.set_tilt(self.device_id, 0)
+        await self.async_update()
```

Comment:
Its unexpected to call `async_update` here... this may be an existing bug that was copied ..  will dig into this

Reply:
Looks like it does update `self._device` in the base entity class.

Thats a bit messy but an existing issue